extensions-core/sample-client-sslcontext/README.md

```diff
@@ -0,0 +1 @@
+This module is a sample implementation of SslContext that will be injected to be used with HttpClient.
```

Comment:
can we also add some docs for SSLClientConfig and other configs needed to work with this module.